cmd/kubelet/app/options/options.go

```diff
@@ -310,9 +305,6 @@ func (f *KubeletFlags) AddFlags(fs *pflag.FlagSet) {
 	f.ContainerRuntimeOptions.AddFlags(fs)
 
 	fs.Var(&f.KubeConfig, "kubeconfig", "Path to a kubeconfig file, specifying how to connect to the API server.")
```

Comment:
would also like to see:
* switch KubeConfig to just be a string
* use `fs.StringVar(&f.KubeConfig, "kubeconfig", f.KubeConfig, ...`
* use `len(s.KubeConfig) > 0` instead of `s.KubeConfig.Provided()`
* do `standaloneMode = len(s.KubeConfig) == 0`